lib/tumblr.js

```diff
@@ -26,8 +26,8 @@ const isArray = require('lodash/isArray');
 const isPlainObject = require('lodash/isPlainObject');
 const omit = require('lodash/omit');
 
-const CLIENT_VERSION = '1.0.0';
-const API_BASE_URL = 'https://api.tumblr.com/v2';
+const CLIENT_VERSION = '3.0.0';
```

Comment:
looks like this never got updated with the actual version in the package.json file :eyes: